gradle.properties

```diff
@@ -1,3 +1,8 @@
 org.gradle.parallel=true
 org.gradle.workers.max=4
 org.gradle.jvmargs=-Xmx4096m -XX:MaxMetaspaceSize=512m
+
+### Override this property to 'true' to enable the end-to-end tests that use docker.
+### This can be done via -Penable.docker.tests=true on the command line, or by
+### setting this property to true in the gradle.properties in your home directory.
+enable.docker.tests=false
```

Comment:
I don't think this is needed, correct? Not present means false. Up to you.

Reply:
I think it's nice to have it documented. But maybe add it as commented-out line.

Reply:
I also like having this as documentation; no reason to comment it out, really. having it "live", I think, makes it clear that it's a real thing, and will be less likely to just bitrot over time. 

That being said, I don't actually care too much, either way.

Reply:
Fine with having it here.